packages/core/src/sagas/sendTypingOnSetSendBoxSaga.js

```diff
@@ -30,8 +30,12 @@ export default function* () {
 
     for (;;) {
       let lastSend = 0;
-      const task = yield takeLatest(SET_SEND_BOX, function* ({ payload: { text } }) {
-        if (text) {
+      const task = yield takeLatest(
+        ({ payload, type }) => (
```

Comment:
inlining this takes away from the readability for me. can it be defined outside the function and given a good name?